Prevent consumers from picking up libomptarget.so from devel/llvm* ports

--- libomptarget/src/CMakeLists.txt.orig	2017-12-06 21:59:15 UTC
+++ libomptarget/src/CMakeLists.txt
@@ -27,5 +27,9 @@ target_link_libraries(omptarget
   ${CMAKE_DL_LIBS}
   "-Wl,--version-script=${CMAKE_CURRENT_SOURCE_DIR}/exports")
 
+if(NOT WIN32)
+  set_target_properties(omptarget PROPERTIES SOVERSION 0)
+endif()
+
 # Install libomptarget under the lib destination folder.
 install(TARGETS omptarget LIBRARY DESTINATION lib${OPENMP_LIBDIR_SUFFIX})
